sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -513,6 +513,23 @@ def get_catalog_df(region_prefix: str) -> pd.DataFrame:
     # Round the prices.
     df['Price'] = df['Price'].round(PRICE_ROUNDING)
     df['SpotPrice'] = df['SpotPrice'].round(PRICE_ROUNDING)
+    gpu_map = {
+        'L4': 24,
+        'A100': 40,
+        'A100-80GB': 80,
+        'A100-40GB': 40,
+        'T4': 16,
+        'P4': 8,
+        'V100': 16,
+        'P100': 16,
+        'K80': 12,
+        '': ''
```

Comment:
sorry why this `''`?

Reply:
This was a mistake on my end, I will have this removed.